### PR TITLE
[Ready] Makes the different ERT hardsuits be different!

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
[Changelogs]
Normal ERT suit has a brighter light and more armor around other then sec hardsuit.
Sec suit now has more armor!
Medical suit can now hold defibs and first aid kits!
Engi suits are now fire proof and protect against heat, like CE!

[why]
Engi suit kinda has to deal with bomb'ed area and heat
Sec is sec
Medical suits already suck if your a race with wings

### CODE REVIEW

**Comment by Ragolution** [comment]
Blue suit is the Commander suit, so I dunno why you decided to make it the 'worst' by comparison.

File: code/modules/clothing/spacesuits/miscellaneous.dm [hunk 1]
```diff
@@ -195,20 +195,27 @@ Contains:
 	item_state = "ert_security"
 	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/ert/sec
 	tauric = TRUE		//Citadel Add for tauric hardsuits
+	armor = list("melee" = 75, "bullet" = 80, "laser" = 80, "energy" = 80, "bomb" = 50, "bio" = 100, "rad" = 100, "fire" = 80, "acid" = 80)
```
Comment (deathride58) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: :thinking:

Comment (Trilbyspaceclone) [reply code/modules/clothing/spacesuits/miscellaneous.dm]: armor = list("melee" = 75, "bullet" = 60, "laser" = 60, "energy" = 60, "bomb" = 55, "bio" = 100, "rad" = 100, "fire" = 80, "acid" = 80)
was the armor it was MENT to be

**Comment by Poojawa** [comment]
All of the ERT suits should be heat/fire proof'd. 

Giving security/leader ERT toggle-able energy shielding might be a fun shakeup as well

**Comment by deathride58** [comment]
The roles in ERT are 100% fluff. You physically cannot pick and choose which role you receive. That's the reason why the hardsuits are all exactly the same, so that getting stuck as a medical ERT when you roll ERT doesn't mean you're permanently doomed to be a healslut for the rest of the round.

**Comment by silicons** [comment]
I really don't see why this is necessary.

**Comment by Trilbyspaceclone** [comment]
@kevinz000 Just making it so that the jobs that you have will have better suited hardsuits is alll
A engi thats not firepoor
A medic that cant hold medical gear
A sec officer that has same armor as a medic
A command suit that is the same as a Engi suit
Just feels and looks funny, why would they not have something to do with their job?

**Comment by Poojawa** [comment]
They would be all identical, standard issue armors. just painted differently is all.

**Comment by Trilbyspaceclone** [comment]
@kevinz000 You seen first hand that this has basic logic behind it.
`ERT suits are fire proof man`
>gets lit on fire and starts to die

**Comment by silicons** [comment]
eh no we'd rather keep them as the same, they shoudl honestly all be immune from burning off from fire/acid but w/e

**Pull request closed without merging**